spec/system/candidate_interface/carry_over/candidate_carries_over_unsubmitted_application_with_a_course_to_new_cycle_spec.rb

```diff
@@ -139,7 +139,6 @@ def when_i_add_a_course
     choose 'Primary (2XT2)'
     click_button t('continue')
 
-    expect(page).to have_content 'You’ve added Primary (2XT2) to your application'
```

Comment:
Might be worth updating these to still assert that the added course is visible on the page (but lower down)?

```ruby
  expect(page).to have_content 'Primary (2XT2)'
```

Reply:
Done ♻️